README.md

```diff
@@ -1,7 +1,59 @@
 # Capacitor Plugins
 
+
 This repository contains the official Capacitor plugins maintained by the Capacitor team. You can find more plugins in the [Capacitor Community](https://github.com/capacitor-community/).
 
+## Watch Duty Fork
+
+This fork contains various changes made by the Watch Duty team to the Camera plugin. To manage this, commits are cherry-picked onto a new branch for each upstream major version change. `main` should always match Capacitor's upstream version of `main`.
+
+> [!IMPORTANT]
+> Check to see if Capacitor has their own major version branch like `17.x`. Sometimes they do, and other times they just use `main`. The examples below use `main` as the upstream branch example.
+
+To implement a minor version change, simply merge the change into our branch:
+
+```sh
+# Update main
+git checkout main
+git fetch upstream
+git merge upstream/master
+
+# Update our branch
+git checkout watchduty-17.x
+git merge main
+```
+
+When a major version is released, create a new branch from `main` and cherry-pick our customization onto that branch:
+
+```sh
+# Update main
```

Comment:
I don't think we need to have a `main` branch locally at all. A simple `git fetch` and then `git checkout -b ... upstream/master` will have the same effect as the multi-step performed here.

Reply:
Good point! Updated.